src/app/core/rest/global.service.ts

```diff
@@ -34,6 +35,10 @@ export class GlobalService {
     return this.httpClient.get<User>(`${this.apiPrefix()}/currentUser`);
   }
 
+  public saveDefaultWorkspace(defaultWorkspace: DefaultWorkspace): Observable<any> {
+    return this.httpClient.put(`${this.apiPrefix()}/workspace`, defaultWorkspace);
```

Comment:
Why is it not defined in the user service if it modifies a user?

Reply:
Because `UserService` has `organizationId` in path and there we don't need it.

Reply:
I am thinking if this is the right approach. We already have two REST methods which work with users but cannot be in user service because they do not need `organizationId`. Maybe it would be worth changing user service to have `organizationId` only on those methods which need it. (Just an idea.)